app/components/post-details-comments.js

```diff
@@ -1,5 +1,13 @@
 import Component from "@ember/component";
+import { inject as service } from '@ember/service';
 
 export default Component.extend({
-
+store: service(),
+commentList: [],
+//this.set using comments with did receive attribute hook
+async init() {
+  this._super(...arguments);
+  const comments = await this.store.query('comment', { postId: query.to.postId });
```

Comment:
@Agathebadia You can get the `postId` from the router service i.e inject the router service and then something like this --> `this.router.currentRoute.params`. You should find the id in there.

Reply:
Thanks! I updated it, but now I have:

```
0: {msg: "PostId is missing from the query", param: "postId", location: "query"}
1: {msg: "PostId should be a uuid", param: "postId", location: "query"}
```

I still have an undefined postId 🤔  I thought postId const would be read properly, but does not look like it...